app/controllers/student_requests_controller.rb

```diff
@@ -70,11 +70,15 @@ def adminview
   
   def updaterequestbyadmin
     @student_request = StudentRequest.find params[:id]
-    @student_request.state = params[:state]
-    @student_request.admin_notes = params[:notes_for_myself]
-    @student_request.notes_to_student = params[:notes_for_student]
-    @student_request.save!
-    flash[:notice] = "The request was successfully updated to " + @student_request.state
+      if(!params[:state] == " ")
```

Comment:
Yeah completely agree with you. I am just saying this is not the right way to do validation... right now by default initiallly it must be showing empty in dropdown right?

I have added some example in comments ... like how it should be? like "Please select the state"